### PR TITLE
Add sendTextViaTyping function

### LINKED ISSUE #2564: typing stops after few seconds
I am using startTyping(phone) and setChatState(phone, 0) in order to maintain the typing ...
i set a timeout function which is executed after 20 seconds which calls setChatState(phone, 2)//paused
The typing starts well but do not maintain the typing for 20 seconds, it just auto stop after few seconds it starts ...

### PR DESCRIPTION
**Pull request by 9cb14c1ec0**
This function works by injecting keystrokes into the browser page, instead of using javascript hooks.  This maintains the typing indicator properly.

Fixes #2564 

To test (it takes a while): `npm install github:9cb14c1ec0/venom#textviatyping`

### CODE REVIEW

File: src/api/layers/sender.layer.ts [hunk 1]
```diff
@@ -362,6 +365,49 @@
...
+
+    let release = await this.typingMutex.acquire();
+    try {
+      let search_element = await this.page.waitForXPath(xpath);
+      // @ts-ignore
+      await search_element.click();
+      let phone_number = to.replace('@c.us', '');
+      await this.page.keyboard.type(' ' + phone_number + '\n', { delay: 200 });
+
+      content = content.replace('\r', '\n');
```
[the remark this answers is not in the document]
Comment (9cb14c1ec0) [reply src/api/layers/sender.layer.ts:389]: Not so.  The replace function replaces all occurrences.

**Comment by levymeireles** [comment]
Is this sendTextViaTyping function already working? I tried to use it today, but it only returns fasle

**Comment by itxtoledo** [comment]
this stopped working, please remove from the lib

**Comment by 9cb14c1ec0** [comment]
> this stopped working, please remove from the lib

Venom is now maintained at https://github.com/venomlib/venom

**Pull request merged**